src/scripts/main.js

```diff
@@ -1,3 +1,255 @@
 'use strict';
 
-// write code here
+// FORM html code
+document.body.insertAdjacentHTML('beforeend', `
+  <form method="/" action="GET" class="new-employee-form">
+  <label>Name:
+    <input name="name" type="text" data-qa="name">
+  </label>
+  <label>Position:
+    <input name="position" type="text" data-qa="position">
+  </label>
+  <label>Office:
+    <select name="office" data-qa="office">
+      <option hidden></option>
+      <option value="Tokyo">Tokyo</option>
+      <option value="Singapore">Singapore</option>
+      <option value="London">London</option>
+      <option value="New York">New York</option>
+      <option value="Edinburgh">Edinburgh</option>
+      <option value="San Francisco">San Francisco</option>
+    </select>
+  </label>
+  <label>Age:
+    <input name="age" type="number" data-qa="age">
+  </label>
+  <label>Salary:
+    <input name="salary" type="number" data-qa="salary">
+  </label>
+  <button type="submit">Save to table</button>
+  </form>
+`);
+
+const thead = document.querySelector('thead');
+const tbody = document.querySelector('tbody');
+const form = document.querySelector('form');
+const rows = tbody.rows;
+
+// Push notification FUNCTION;
+function pushNotification(message, type) {
+  const errorTitle = `Form was't added !!!`;
+  const successTitle = `Form was added`;
+  const notifyErrors = {
+    name: 'Name field should have not less then 4 character',
+    position: 'Position field should have not less then 4 character',
+    office: 'Office field should be selected',
+    age: 'Age field should be from 18 to 90',
+    salary: 'Salary field should be from 90$',
+  };
+  const successDescription = `Employee was added in the end of the table!\n `
+    + `To edit the data double click on a cell!`;
+
+  const title = type === 'success' ? successTitle : errorTitle;
+  const body = document.querySelector('body');
+
+  body.insertAdjacentHTML('beforeend', `
+    <div class="notification" class=${type} data-qa="notification">
+      <h2 class="title">${title}</h2>
+    </div>
+  `);
+
+  const div = document.querySelector('.notification');
+
+  if (message.length > 0) {
+    let numberOfParagraph = 1;
+
+    for (const error of message) {
+      const p = document.createElement('p');
+
+      p.innerText = `${numberOfParagraph}. ${notifyErrors[error]}`;
+      div.append(p);
+      numberOfParagraph++;
+    }
+  } else {
+    const p = document.createElement('p');
+
+    p.innerText = successDescription;
+    div.append(p);
+  }
+
+  div.classList.add(type);
+  div.style.top = '450px';
+
+  setTimeout(() => {
+    document.querySelectorAll('.notification').forEach(x => x.remove());
+  }, 6000);
+};
+
+// Sorting tables FUCTION
+function sortElemInTwoWays(element, start, end) {
```

Comment:
It's better to use more descriptive function names. Consider renaming 'sortElemInTwoWays' to 'sortTableColumn' or something similar.